### ISSUE TITLE
Keybinding to load ESS

### ISSUE THREAD

**Issue by TheBB** [syl20bnr/spacemacs#3290]
At the moment, ESS must be loaded manually using the binding `SPC e s s`. Since #3223 `SPC e s` is taken by flycheck. Since the original binding for ESS feels out of place in Spacemacs' neatly organized hierarchy I feel it would be better to change it. Ideas?

**Comment by izahn** [comment]
Honestly I think we should just load ess on startup if the `ess` layer is enabled. It takes less than a second on my 5 year old $300 laptop:

```elisp
(add-to-list 'load-path "/home/izahn/Development/dotemacs/elpa/ess-20151008.1451/lisp")

(defmacro measure-time (&rest body)
  "Measure the time it takes to evaluate BODY."
  `(let ((time (current-time)))
     ,@body
     (message "%.06f" (float-time (time-since time)))))

(measure-time (require 'ess-site))
;; 0.790182
```
Granted we can't afford to spend this much time on each layer, but I'm more than willing to wait an extra fraction of a second to have ESS loaded and ready to go. I think doing this will also simplify the code needed to set up ESS in spacemacs, making it easier to fix when things go wrong. I recently reported https://github.com/syl20bnr/spacemacs/issues/3313 after spending some time going through the ESS layer code. The code is hard to follow (at least it was for me, I only have basic elisp skills) in part because of all the workarounds to avoid loading ESS on startup.

**Comment by izahn** [comment]
I've opened https://github.com/syl20bnr/spacemacs/pull/3342 to fix this issue. The fix eliminates the need to manually load ESS, thereby eliminating the keybinding conflict, without increasing load times.
